pkg/apis/core/v1beta1/doc.go

```diff
@@ -9,7 +9,7 @@
 // +k8s:defaulter-gen=TypeMeta
 // +k8s:protobuf-gen=package
 
-//go:generate gen-crd-api-reference-docs -api-dir . -config ../../../../hack/api-reference/core-config.json -template-dir ../../../../hack/api-reference/template -out-file ../../../../docs/api-reference/core.md
+//go:generate gen-crd-api-reference-docs -api-dir github.com/gardener/gardener/pkg/apis/core/v1beta1 -config ../../../../hack/api-reference/core-config.json -template-dir ../../../../hack/api-reference/template -out-file ../../../../docs/api-reference/core.md
```

Comment:
For consistency reason, don't we have to adapt all `//go:generate gen-crd-api-reference-docs -api-dir` usages in the repo?

Reply:
So far the other docs files were not affected (at least to my knowledge). Should we adapt them anyway?

Reply:
I haven't read the referenced issue but for consistency reasons - I would answer "yes". Because when tomorrow someone asks why only single occurrence uses `//go:generate gen-crd-api-reference-docs -api-dir <package>` and the rest use `//go:generate gen-crd-api-reference-docs -api-dir .`, I won't be able to answer.

And maybe in this way we could prevent the issue from occurring for the other API groups? Or is there other reason why only the core v1beta1 API group is affected?